packages/split-button/src/SplitButton.story.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { MouseEvent, useRef } from 'react';
+
+import { Size } from '@leafygreen-ui/button';
+import Icon, { glyphs } from '@leafygreen-ui/icon';
+import { storybookArgTypes, StoryMeta } from '@leafygreen-ui/lib';
+import { MenuItem } from '@leafygreen-ui/menu';
+
+import { Align, Justify, SplitButton, SplitButtonProps, Variant } from '.';
+
+export default StoryMeta({
+  title: 'Components/SplitButton',
+  component: SplitButton,
+  args: {
+    label: 'label',
+    variant: Variant.Default,
+    align: Align.Bottom,
+    justify: Justify.End,
+    spacing: 6,
+    menuItems: (
```

Comment:
I'm not convinced this is the best pattern still. I wonder if it should be an array of `<MenuItem />` components. Or at least if we can be opinionated about if it's a fragment or an array and not solve for both. You missed the RFC process @spark33  - what do you think?

Reply:
My issue with an array is that it makes it incompatible with Menu. 

If a developer currently has a Menu that they want to change to a SplitButton, they can just copy/paste their MenuItems as they were in Menu. If we require an array they'll need to change (albeit slightly) how they pass the items in.

Reply:
Yeah I'm with you @bruugey - I think we should stray from parsing children to dictate behavior in all cases. I'm okay with introducing a small migration cost to the user to avoid parsing children. I'd also be okay with accepting an array of "config" for MenuItem props, and we can create the MenuItem instances ourselves